coffee_httpd/src/httpd/server.rs

```diff
@@ -13,6 +13,7 @@ use std::sync::Arc;
 use serde_json::Value;
 use tokio::sync::Mutex;
 
+use super::utils::*;
 use coffee_core::coffee::CoffeeManager;
 use coffee_lib::plugin_manager::PluginManager;
```

Comment:
```suggestion
use coffee_core::coffee::CoffeeManager;
use coffee_lib::plugin_manager::PluginManager;

use super::utils::*;
```

I need to write down some rules for the import for now we move down it